cluster-provision/gocli/opts/prometheus/manifests/prometheus/prometheus-prometheus.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: monitoring.coreos.com/v1
+kind: Prometheus
+metadata:
+  labels:
+    app.kubernetes.io/component: prometheus
+    app.kubernetes.io/name: prometheus
+    app.kubernetes.io/part-of: kube-prometheus
+    app.kubernetes.io/version: 2.26.0
+    prometheus: k8s
+  name: k8s
+  namespace: monitoring
+spec:
+  alerting:
+    alertmanagers:
+    - apiVersion: v2
+      name: alertmanager-main
+      namespace: monitoring
+      port: web
+  externalLabels: {}
+  image: quay.io/prometheus/prometheus:v2.26.0
```

Comment:
Can you update the monitoring manifests to match the latest versions that were updated in this PR - https://github.com/kubevirt/kubevirtci/pull/1228